source/solvers/vof.cc

```diff
@@ -241,6 +241,28 @@ void
 VolumeOfFluid<dim>::attach_solution_to_output(DataOut<dim> &data_out)
 {
   data_out.add_data_vector(this->dof_handler, this->present_solution, "phase");
+
+  if (this->simulation_parameters.multiphysics.continuum_surface_force &&
+      simulation_parameters.surface_tension_force.output_VOF_auxiliary_fields)
+    {
+      std::vector<DataComponentInterpretation::DataComponentInterpretation>
+        pfg_component_interpretation(
+          dim + 1, DataComponentInterpretation::component_is_scalar);
```

Comment:
Why does this have dim+1 component if it is the gradient ? Should it not have dim components?

Reply:
Corrected.